rust/src/lib/ovs.rs

```diff
@@ -176,7 +182,25 @@ impl MergedOvsDbGlobalConfig {
             if let Some(ex_ids) = desired.external_ids.as_ref() {
                 if ex_ids.is_empty() {
                     external_ids.clear();
+                    if let Some(current_bridge_mappings) = current_ovn_mappings
+                    {
+                        if !current_bridge_mappings.is_empty() {
+                            external_ids.insert(
+                                OVN_BRIDGE_MAPPINGS.to_string(),
+                                Some(ovn_bridge_mappings_to_string(
+                                    current_bridge_mappings,
+                                )),
+                            );
+                        }
+                    };
                 } else {
+                    if ex_ids.get(OVN_BRIDGE_MAPPINGS).is_some() {
+                        const INVALID_EXTERNAL_IDS_KEY: &str = "Cannot use the `ovn-bridge-mappings` external_ids key directly. Please use ovs_db.ovn-bridge-mappings API instead";
```

Comment:
Please change `ovs_db.ovn-bridge-mappings` to `ovn.bridge-mappings`.

Please also break long line with `\`

Reply:
Good catch.